CHANGELOG.asciidoc

```diff
@@ -131,6 +132,8 @@ https://github.com/elastic/beats/compare/v6.5.0...6.x[Check the HEAD diff]
 
 *Metricbeat*
 
+- Deprecate field `metricset.rtt`. Replaced by `event.duration` which is in nano instead of micro seconds.
+
```

Comment:
There's no update to the definition for `metricset.rtt` to mark it as deprecated. I think it would make sense to add it in this PR. WDYT?

Reply:
Good point, added it to the fields definition. We have this in a few other places but still need to add the generation for it.